frontend/src/component/feature/FeatureView/FeatureViewHeader.tsx

```diff
@@ -49,8 +49,16 @@ const NewStyledHeader = styled('div')(({ theme }) => ({
     backgroundColor: 'none',
     marginBottom: theme.spacing(2),
     borderBottom: `1px solid ${theme.palette.divider}`,
+    containerType: 'inline-size',
 }));
 
+const onNarrowHeader = (css: object) => ({
+    '@container (max-width: 650px)': css,
+    '@supports not (container-type: inline-size)': {
+        '@media (max-width: 700px)': css,
+    },
+});
```

Comment:
Use container queries to decide when to stack actions over tabs.

Doesn't use the predefined breakpoints because:
- for container queries: It doesn't make sense for containers to use predefined screen widths
- for the media query: 650px ('sm') is too small (causes wrapping or overlap) and 900px ('md') feel too wide a place to start wrapping